guides/v2.2/cloud/project/project-conf-files_magento-app.md

```diff
@@ -39,33 +39,33 @@ The `build` property determines what happens by default when building the projec
 
 ```yaml
 build:
-    flavor: composer
+  flavor: composer
```

Comment:
PR #5039 will blow this away

Our YAML files may or may not conform to our linter. It depends on the YAML we refer to in the development repo.
Remove all changes to this file.

Reply:
Done.